test/eyeshade/owners.integration.test.js

```diff
@@ -186,6 +189,8 @@ test('eyeshade PUT /v1/owners/{owner}/wallet with uphold parameters', async t =>
   t.is(Array.isArray(wallet['possibleCurrencies']), true, 'get wallet returns currencies we could create a card for')
   t.is(wallet['possibleCurrencies'].indexOf('BAT') !== -1, true, 'wallet can have a BAT card')
```

Comment:
consider using dot notation, and deconstruct where possible. this is a great example of where it makes a lot of sense.

Reply:
:+1: we should switch this file over to that convention